nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -560,6 +560,8 @@ def test_plot_prob_atlas():
     # Testing the 4D plot prob atlas with colormap
     plot_prob_atlas(img, view_type='filled_contours', colorbar=True)
     plt.close()
+    # threshold=None
+    plot_prob_atlas(img, threshold=None)
```

Comment:
> BTW, I have added plt.close everywhere to save testing time and it was
> efficient. Do you think that we should configure nose to close all figures
> after each test?

I am worried that I will create more infrastructure that is opaque to non
expert contributors. I prefer simple solutions like this.